docs/authentication/saml/authentication-flows.mdx

```diff
@@ -40,16 +43,20 @@ A bad actor could also can intercept the IdP response and replace it with anothe
 Even if the IdP-initiated flow is considered insecure, Clerk tries to mitigate those flaws in order to protect and offer the best possible security for your application and users.
 
 #### Unsolicited `InResponseTo` attribute
```

Comment:
This is a heading under the "Risks of IDP-initiated flow" section. Is "Unsolicited `InResponseTo` attribute" a risk of an IDP-initiated flow?

Reply:
Hey, "Unsolicited InResponseTo attribute" is one of the ways we try to mitigate the risks of an IDP-initiated flow. Same goes for the rest of the following headings